haskell-language-server.cabal

```diff
@@ -373,7 +373,7 @@ test-suite func-test
     , tasty-golden
     , tasty-rerun
 
-  hs-source-dirs:     test/functional plugins/tactics/src
+  hs-source-dirs:     test/functional plugins/tactics/src plugins/hls-eval-plugin/test
```

Comment:
I think what @pepeiborra meant is not only moving code files into inner directory but also moving (and adding) test-suite to `hls-eval-plugin.cabal` like tactic plugin.